contrib/cf-remote/cf_remote/packages.py

```diff
@@ -117,6 +117,13 @@ def add_tags_from_filename(self, filename):
         else:
             self.add_tag("client")
             self.add_tag("agent")
+        parts = filename.split(".")
+        if "x86_64" in parts:
+            self.add_tag("x86_64")
+            self.add_tag("64")
+        if "amd64" in parts:
+            self.add_tag("amd64")
+            self.add_tag("64")
```

Comment:
Should this be part of the previous commit?

Reply:
Fixed.